.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Generate and publish Android Matrix SDK documentation
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  docs:
+    name: Docs
```

Comment:
I would add a more detailed job name like `Generate Android sdk documentation`
and a shorter  .yml file name like `Documentation`

Reply:
done